### PR TITLE
Go to waypoint

### LINKED ISSUE #30: Auto waypoint alignment
Automatically, using pathplanner's newer trajectory generator, go to a set of preset waypoints with button press
eventually should incorperate with certain positions also getting the shooter into those preset shots (podium, subwoofer)
side dependant waypoints

### PR DESCRIPTION
**Pull request by icai0**
goes to the closest waypoint (speaker, amp, source)
more waypoints can be added later, along with changing the restricted zones in pathplanner

### CODE REVIEW

File: ControlScheme.json [hunk 1]
```diff
@@ -243,5 +243,12 @@
       "Button": 7
     },
     "buttonbox"
+  ],
+  "goToAmp": [
+    5,
+    {
+      "Button": 1
+    },
+    "temporary"
```
Comment (lmaxwell24) [inline ControlScheme.json]: nuh uh use the actual control we already added for that exact feature

File: operatorinterface.py [hunk 2]
```diff
@@ -222,3 +222,4 @@ def getPOVBindingOfName(name: str) -> typing.Tuple[Joystick, int, int]:
 
         self.elevatorJogUp = getButtonBindingOfName("elevatorJogUp")
         self.elevatorJogDown = getButtonBindingOfName("elevatorJogDown")
+        self.goToAmp = getButtonBindingOfName("goToAmp")
```
Comment (lmaxwell24) [inline operatorinterface.py]: no :(

**Review by lmaxwell24 — CHANGES_REQUESTED** [review]

File: constants.py [hunk 3]
```diff
@@ -879,3 +881,28 @@
 kAlignAnglePGain = 1.2
 kAlignAngleIGain = 0
 kAlignAngleDGain = 0
+
+kPathfindingConstraints = PathConstraints(
+    kMaxWheelLinearVelocity / 2,
```
Comment (lmaxwell24) [inline constants.py:906]: eventually we can go faster but not rn is fine

**Review by lmaxwell24 — DISMISSED** [review]

**Comment by lmaxwell24** [comment]
legit github backends are being silly and thats kinda it
if theres a second approve I will force merge

**Review by LandonBayer — DISMISSED** [review]

**Pull request merged**